src/test/integ/buffer-write-cmd.test.ts

```diff
@@ -15,6 +16,22 @@ import {
 
 describe("BufWriteCmd integration", () => {
     const testFiles: Uri[] = [];
+    const cleanupCallbacks: (() => Promise<void>)[] = [];
+
+    const cleanupFileCallback = async (uri: Uri) => {
```

Comment:
nit: no reason to mention "callback" in the name.
```suggestion
    const cleanupFile = async (uri: Uri) => {
```

Reply:
Ack, updated in the latest revision